src/tests/utils/ApiTestSteps.ts

```diff
@@ -243,3 +243,27 @@ export async function validateTxMAEventData(keyList: any): Promise<any> {
 			});
 	}
 }
+
+export async function validateJwtToken(jwtToken: any): Promise<void> {
+	const [rawHead, rawBody, signature] = jwtToken.split(".");
+
+	validateRawHead(rawHead);
+	validateRawBody(rawBody);
+}
+
+function validateRawHead(rawHead: any): void {
+	const decodeRawHead = JSON.parse(jwtUtils.base64DecodeToString(rawHead.replace(/\W/g, "")));
+	expect(decodeRawHead.alg).toBe("ES256");
+	expect(decodeRawHead.typ).toBe("JWT");
+}
+
+function validateRawBody(rawBody: any): void {
+	const decodedBody = JSON.parse(jwtUtils.base64DecodeToString(rawBody.replace(/\W/g, "")));
+	expect(decodedBody.jti).toBeTruthy();
+
+	// Strength Score
```

Comment:
[nit] are these comments needed? We can see it's `strengthScore` and `validityScore` being accessed